lib/dor/services/workflow_service.rb

```diff
@@ -519,7 +526,11 @@ def count_objects_in_step(workflow, step, type, repo)
       def workflow_resource_method(uri_string, meth = 'get', payload = '', opts = {})
         with_retries(:max_tries => 2, :handler => @@handler, :rescue => workflow_service_exceptions_to_catch) do |attempt|
           @@logger.info "[Attempt #{attempt}] #{meth} #{workflow_resource.url}/#{uri_string}"
-          if %w[get delete].include?(meth)
+          if meth == 'get'
+            fail NotImplementedError, "GET does not support extra headers: #{opts}" unless opts.length == 0
```

Comment:
i'm not sure how strict the conventions are when there's gray area between the two, but either seems reasonable.  it's behavior that you might expect to be implemented, but it's not actually implemented, and it's not a categorically crazy argument for this sort of method.